mix.exs

```diff
@@ -34,24 +34,21 @@ defmodule Warehouse.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:amqp, "~> 2.0", override: true},
-      {:bottle, github: "system76/bottle", ref: "62b2704"},
-      {:broadway_rabbitmq, "~> 0.6.0"},
-      {:credo, "~> 1.3", only: [:dev, :test]},
-      {:decimal, "~> 1.9.0", override: true},
-      {:decorator, "~> 1.2"},
-      {:dialyxir, "~> 1.0", only: [:dev, :test], runtime: false},
-      {:ecto_sql, "~> 3.5"},
-      {:ex_aws_sqs, "~> 3.2"},
-      {:ex_aws, "~> 2.1.6"},
```

Comment:
Library cleanup, we don't use AWS SQS anymore so don't need them. By upgrading bottle we also can run the app on latest Erlang and Elixir that give us some nice goodies.